accepted/future-releases/small-features-21Q1/feature-specification.md

```diff
@@ -113,3 +113,30 @@ on the `int` class, which will work similarly to `>>`, but will zero-extend inst
 At that point, the `>>>` operator on `int` must be **valid in constant and potentially constant expressions**, so `0x40 >>> 3` is a compile-time constant expression with the value `8`, and `const C(int x) : y = 0xFFFFFFFF >>> x;` is valid (although potentially throwing if `x > 63`) as a constant constructor.
 
 Backends may want to optimize this to use available bitwise shift operations (like `>>>` in JavaScript), and intrinsify the function if possible. This can be done at any later time, though.
+
+## Mixed Mode Programs
+
+Libraries using a language version prior to the introduction of these features (opted out libraries) 
+interact with libraries using those features (opted in libraries) as follows.
+
+In an opted out library:
+* It is a compile-time error to declare an operator method named `>>>`, or if to have `e1 >>> e2` as an expression.
+* It is a compile-time error to contain an explicit type argument on an annotation constructor invocation.
```

Comment:
The meaning of the word `contain` isn't obvious here. Maybe `..error if a type argument list occurs..`?